docs/ECOS.md

```diff
@@ -138,10 +166,10 @@ ECI Distribution Specification will closely follow [OCI Distribution Specificati
 
 The goals for the ECI Distribution Specification is to address the following aspects of ECI distribution:
 
-  * Mapping of symbolic names to content-addressable handles
-  * Standardizing operations on metadata for collection of ECIs stored as a group
-  * Establishing trust model around managing software supply chain all the way to the final ECI
-  * Effective download of individual binary blobs corresponding to the ECI components
-  * Effective upload of individual binary blobs corresponding to the ECI components
+* Mapping of symbolic names to content-addressable handles
+* Standardizing operations on metadata for collection of ECIs stored as a group
+* Establishing trust model around managing software supply chain all the way to the final ECI
+* Effective download of individual binary blobs corresponding to the ECI components
+* Effective upload of individual binary blobs corresponding to the ECI components
```

Comment:
I question if we need a distribution specification at all. I discussed this at length with @rvs . The [OCI Distribution Spec](https://github.com/opencontainers/distribution-spec) is directly compatible with what we need for ECI distribution. It also makes every container registry under the sun (docker hub, ECR, quay.io, Harbor, etc.) usable, and greatly simplifies the EVE pull mechanism: we only need to support one kind of image pull, and we can use the moby/containerd libraries for it, eliminating any custom work. It also includes all of the image hashing and validation, and we even could adopt TUF/Notary if we want to with zero additional work (assuming we actually understand it :-) ).